### ISSUE TITLE
Handling commas in checkboxes component

### ISSUE THREAD

**Issue by gabrielrobert** [gitana/alpaca#577]
Given this configuration, anything will work fine. `should be selected` checkbox will be checked.

https://jsfiddle.net/o7dkk2s8/1/

```
$("#target").alpaca({
    "data": "should be selected, cookie, drink",
    "schema": {
        "type": "string",
        "enum": ["should be selected", "chips", "cookie", "drink"]
    },
    "options": {
        "type": "checkbox",
        "label": "What would you like with your order?",
        "optionLabels": ["should be selected", "Potato Chips", "A Cookie", "Soft Drink"]
    }
});
```

But if my data contains a comma, AlpacaJS cannot handle it properly. Example:

https://jsfiddle.net/o7dkk2s8/3/

```
$("#field5").alpaca({
    "data": "should be, selected, cookie, drink",
    "schema": {
        "type": "string",
        "enum": ["should be, selected", "chips", "cookie", "drink"]
    },
    "options": {
        "type": "checkbox",
        "label": "What would you like with your order?",
        "optionLabels": ["should be selected", "Potato Chips", "A Cookie", "Soft Drink"]
    }
});
```

`should be, selected` should be selected, but it's not. Any ideas?

**Comment by ambischof** [comment]
Try making it an array instead: 

```
$("#target").alpaca({
    "data": ["should be, selected", "cookie", "drink"],
    "schema": {
        "type": "array",
        "enum": ["should be, selected", "chips", "cookie", "drink"]
    },
    "options": {
        "type": "checkbox",
        "label": "What would you like with your order?",
        "optionLabels": ["should be selected", "Potato Chips", "A Cookie", "Soft Drink"]
    }
});```

**Comment by mibewh** [comment]
This issue has been fixed for new builds of Alpaca: e5e4505. The issue was related to how data is parsed for multiple select ListFields.
